content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md

```diff
@@ -38,53 +38,13 @@ Let's say you have a Deployment containing of a single `nginx` replica
 
 {{< codenew file="service/pod-with-graceful-termination.yaml" >}}
 
-```yaml
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: nginx-deployment
-  labels:
-    app: nginx
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: nginx
-  template:
-    metadata:
-      labels:
-        app: nginx
-    spec:
-      terminationGracePeriodSeconds: 120 # extra long grace period
-      containers:
-      - name: nginx
-        image: nginx:latest
-        ports:
-        - containerPort: 80
-        lifecycle:
-          preStop:
-            exec:
-              # Real life termination may take any time up to terminationGracePeriodSeconds.
-              # In this example - just hang around for at least the duration of terminationGracePeriodSeconds,
-              # at 120 seconds container will be forcibly terminated.
-              # Note, all this time nginx will keep processing requests.
-              command: [
-                "/bin/sh", "-c", "sleep 180"
-              ]
+{{< codenew file="service/explore-graceful-termination-nginx.yaml" >}}
 
----
+Now create the Deployment Pod and Service using the above files:
 
-apiVersion: v1
-kind: Service
-metadata:
-  name: nginx-service
-spec:
-  selector:
-    app: nginx
-  ports:
-    - protocol: TCP
-      port: 80
-      targetPort: 80
+```shell
+kubectl apply -f pod-with-graceful-termination.yaml
+kubectl apply -f explore-graceful-termination-nginx.yaml
```

Comment:
Please use the URLs of the examples, not the local path. Have a look at other pages that use `codenew` to see how it's done.

Reply:
Hi there, @chirag0002! 👋

Could you please update these `kubectl apply` commands to use the URLs for the sample files instead of the local paths, as requested previously by @sftim?

One example of how this is done can be seen in the [Configure Minimum and Maximum CPU Constraints for a Namespace](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/#create-a-limitrange-and-a-pod) page: https://github.com/kubernetes/website/blob/172f17e16146394c6129deabfa30dd2a152dded5/content/en/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md?plain=1#L50-L56